lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -333,8 +333,6 @@ CALL_TICF_CRI:
   response:
     type: process
     lambda: call-ticf-cri
-    lambdaInput:
-      journeyType: main
```

Comment:
Do we need removal of lambdaInput from journeyEngineStepFunction.asl.json?

Reply:
Ooo, good shout